src/store/utils/observeStore.ts

```diff
@@ -0,0 +1,19 @@
+import { Store } from 'redux';
+
+// Taken from https://github.com/reduxjs/redux/issues/303#issuecomment-125184409
+export default function observeStore(store: Store, select: Function, onChange: Function): Function {
```

Comment:
Still needed? It's not being used at the moment.

Reply:
Sure, I can remove it, we can always add it back if we need to